### DISCUSSION TITLE
Replaced Wallet - Incorrect Wallet Balance Showing?

### DISCUSSION THREAD

**Discussion by olimits7** [btcpayserver/btcpayserver#5887] [Technical Support]
Hi, I needed to replace my current wallet with a new wallet that I created and then send all my funds to.  However, in BTCPayServer the wallet is showing a zero balance, however, when I view the wallet in Electrum I can see my correct balance.  Why is BTCPayServer not showing the correct wallet balance? Ty.

**Comment by dennisreimann** [comment]
Can you ensure the wallet is indeed the same by comparing the fingerprint? If it is the same, tthe prroblem might be a missing sync. As an admin you can rescan the wallet - does tthat yield the correct balance and associated ttransactions?

**Reply by olimits7** [reply]
I actually use Voltage for my BTCPayServer.  I reached out to them and it looks like they disable the "Rescan Wallet" function because that would rescan everyone using BTCPayServer on Voltage because it's a shared platform.  So there is no other way to resolve this issue, only the "Rescan Wallet" option?

**Reply by dennisreimann** [reply]
You could also reimport it, but as you are managing that wallet in Electrum and want to replace it anyways – what's the point?

**Reply by olimits7** [reply]
Yeah, but I like being able to see the current balance show in BTCPayServer since I usually don't have Electrum open.  So you mean to do the "Import wallet file" option when I replace wallet?  That's what I did to connect to the new wallet I created where I transferred the funds to from the old wallet I had. Ty.
